src/game.zig

```diff
@@ -904,6 +1002,9 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 						movementSpeed = @max(movementSpeed, 5.5);
 						movementDir[2] += 5.5;
 					}
+				} else if(Player.super.climbing) {
+					movementSpeed = @max(movementSpeed, Player.currentClimbSpeed*0.7);
```

Comment:
This feels wrong. When jumping next to a climbable block you should be able to keep your jumping speed, slowly slowing down to the ladder speed due to gravity.

Furthermore using `@min` and `@max` in collision code is always bad, since it has the ability to cut your velocity instantly as soon as you touch a certain block.
It's better to use friction to slow it down slowly instead.